src/composite/concentrator.jl

```diff
@@ -54,8 +54,9 @@ function apply!(r::AbstractRegister, c::Concentrator)
     return r
 end
 
-mat(c::Concentrator{N, T, <:AbstractBlock}) where {N, T} =
-    error("Not implemented, post an issue if you really need it.")
+function mat(c::Concentrator{N, T, <:AbstractBlock}) where {N, T}
+    mat(PutBlock{N}(c.content, c.locations))
```

Comment:
Why we didn't use this before? I thought concentrate should be some rank-3 tensor?

Reply:
I was considering about performance issues, now I think this is good enough. BTW: Since it is an operator, it should not be a rank-3 tensor...

Reply:
fine, I'll wait for CI then.